src/components/EditListResultViewer/EditListResultViewer.tsx

```diff
@@ -69,7 +69,7 @@ export const EditListResultViewer:FC<EditListResultViewerProps> = (
       entityTypeDataSource={getAsyncEntityType}
       visibleColumns={visibleColumns}
       onSetDefaultColumns={() => {}}
-      height={500}
+      height={525}
```

Comment:
I'm not sure why this is the magic number; normally, we have `500` as the hardcoded height for the result viewer. However, with `500`, the scrollbar is cut off below this threshold, so 🤷 

And I can't imagine the scrollbar would be more than this margin